lib/config/configuration.go

```diff
@@ -263,6 +267,22 @@ type IntegrationConfAccessGraphAWSSync struct {
 	Role string
 }
 
+// IntegrationConfAzureOIDC contains the arguments of
+// `teleport integration configure azure-oidc` command
+type IntegrationConfAzureOIDC struct {
+	// ProxyPublicAddr is the publicly-reachable URL of the Teleport Proxy.
+	// It is used as the OIDC issuer URL, as well as for SAML URIs.
+	ProxyPublicAddr string
+
+	// AuthConnectorName is the name of the SAML connector that will be created on Teleport side.
+	AuthConnectorName string
+
+	// AccessGraph is a flag indicating that access graph integration is requested.
+	// When this is true, the integration script will produce
+	// a cache file necessary for TAG synchronization.
+	AccessGraph bool
```

Comment:
nit: I'd rename it to `AccessGraphEnabled` or something like that, but I also don't mind the current version.